AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -519,8 +520,17 @@ - (void)accessDataSourceSynchronously:(BOOL)synchronously withBlock:(dispatch_bl
     for (NSUInteger i = 0; i < rowNum; i++) {
       NSIndexPath *indexPath = [sectionIndex indexPathByAddingIndex:i];
       ASCellNodeBlock nodeBlock = [_dataSource dataController:self nodeBlockAtIndexPath:indexPath];
+      
+      // When creating a node, make sure to pass along the current display traits so it will be laid out properly
+      ASCellNodeBlock nodeBlockPropagatingDisplayTraits = ^{
+        ASCellNode *cellNode = nodeBlock();
+        id<ASEnvironment> environment = [self.environmentDelegate dataControllerEnvironment];
+        ASEnvironmentStatePropagateDown(cellNode, [environment environmentState].traitCollection);
```

Comment:
would be nice to have something like `environment.traitCollection`, even if it's brought in by a category.